doc/mgr/dashboard.rst

```diff
@@ -259,6 +270,18 @@ commands::
 
   $ ceph dashboard ac-user-create <username> <password> administrator
 
+Accessing the Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can now access the dashboard using your (JavaScript-enabled) web browser, by
+pointing it to any of the host names or IP addresses and the selected TCP port
+where a manager instance is running: e.g., ``httpS://<$IP>:<$PORT>/``.
+
+You should then be greeted by the dashboard login page, requesting your
+previously defined username and password. Select the **Keep me logged in**
```

Comment:
Do we still have `**keep me logged in** ` checkbox?

Reply:
No, I don't think it exists anymore.

Reply:
Good spotting, will remove this part.